src/style.css

```diff
@@ -1,3 +1,72 @@
-body {
+html {
+  font-family: "Roboto", sans-serif;
+  --normal-color: #000;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+body,
+ul,
+li,
+header {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  --active-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.moyo-logo {
+  display: flex;
+  align-items: center;
+}
+
+.nav__list {
+  list-style-type: none;
+  display: flex;
+  align-items: center;
+  text-transform: uppercase;
+}
+
+.nav__item {
+  position: relative;
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin: 0;
+}
+
+.nav__link {
+  display: flex;
+  height: 60px;
+  align-items: center;
+  text-decoration: none;
+  color: var(--normal-color);
+
+}
+
+.nav__link:hover {
+  color: var(--active-color);
+}
+
+.is-active {
+  color: var(--active-color);
+}
+
+.nav__link.is-active::after {
+  display: block;
+  content: "";
+  height: 4px;
+  width: 100%;
+  background: var(--active-color);
+  position: absolute;
+  border-radius: 8px;
```

Comment:
border-radius should be half of width/height. If you set bigger value it is not a good practice.